indico/modules/events/registration/client/js/form/fields/DateInput.jsx

```diff
@@ -199,7 +200,7 @@ export const dateSettingsInitialData = {
   timeFormat: null,
 };
 
-export function DateSettings() {
+export function DateSettings(props) {
```

Comment:
are any props being passed to this component? if not, you can omit this

Reply:
I need the settings in args to pass them back to the plugin, if I don't, the plugin won't have access to them.